packages/smart-wallet/typedoc.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": [
+        "../../typedoc.base.json"
+    ],
+    "entryPoints": [
+        "src/offers.js",
+        "src/smartWallet.js",
+        "src/types.js",
+        "src/walletFactory.js",
```

Comment:
walletFactory doesn't subsume the others as entry points?

(interesting that we don't have a "main" key in smart-wallet/package.json)

Reply:
Nope. I added these after getting warnings like,
```
[warning] InvitationSpec, defined in ./packages/smart-wallet/src/invitations.js, is referenced by offers.OfferSpec.__type.invitationSpec but not included in the documentation.
[warning] Petname, defined in ./packages/smart-wallet/src/types.d.ts, is referenced by smartWallet.BrandDescriptor.__type.petname but not included in the documentation.
[warning] MakeOfferWatcher, defined in ./packages/smart-wallet/src/offerWatcher.js, is referenced by smartWallet.ExecutorPowers.__type.makeOfferWatcher but not included in the documentation.
[warning] BrandDescriptor, defined in ./packages/smart-wallet/src/walletFactory.js, is referenced by walletFactory.makeAssetRegistry.makeAssetRegistry.__type.get.__type.__type but not included in the documentation.
```